index.rst

```diff
@@ -10,20 +10,849 @@
+Therefore, we consider here three separate states of functionality for the observatory in the event of an outage as defined below.
+
+State 1:
+   The observatory is able to safely continue standard survey operations with minimal functionality to evaluate science data quality in real time.
+   Image display is still occurring because the CVT is hosted on the summit-based diagnostic cluster and observers can visually inspect raw images and images with minimal instrument signature removal.
+   Low-level calculations and analysis will go into the camera database and the EFD.
+State 2:
+   As above, with the addition of the Rapid Analysis framework to support operations, scheduler input, QA analyses etc.
+State 3:
+   Full operations, including all processing that is planned to be performed at the USDF, such as Alert Processing, with transfer of diagnostic information back to the summit.
+
+Maintaining State 3 in the event of a network outage means moving all Alert Processing infrastructure to the summit.
+This is not practical for many reasons, nor is it a requirement, and is therefore not discussed further.
+
+In the event of a network failure between summit and base, the observatory would at most be able to achieve State 1.
+Because no Rapid Analysis support will be available from the base, any (non-AOS) image-based calculations will not be performed and therefore it is possible that certain engineering tests will not be able to be run, and (potentially) certain inputs to the Scheduler may not arrive.
+
+If we consider that the Camera Diagnostic Cluster could perform some of the tasks considered in State 2, for example, a subset of Rapid Analysis is required (which we refer to as rapid-analysis-critical) to remain functional in the event of an outage, this requires a very significant increase in functionality.
+
+- DM tooling must be installed and maintained on the Camera Diagnostic Cluster
+- Rapid-analysis-critical must be developed and deployed, with the ability to only focus on a subset of detectors, and/or metrics
+- The database containing the output must be hosted on the summit, then replicated outwards
+
+Note that the full output of Rapid Analysis cannot be computed due to the limited compute power.
+
+This committee suggests that if Antu does need to stay at the base, then a step-wise approach where the infrastructure for scenario 1 gets implemented prior to significant effort being put into scenario 2, if deemed appropriate.
+The preferred solution is to move the Antu servers to the summit.
+
+.. _antu_at_summit:
+
+Antu at the Summit (Proposed Change)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Another possibility which has been considered by this group is to relocate the Antu servers to the summit, even if it means reducing the cluster size in Chile and increasing the capability at the USDF.
```

Comment:
If you agree to merge this session with the previous session I think you can remove this phrase or merge the "even if it means reducing the cluster size in Chile and increasing the capability at the USDF." part with the proposal given above, e.g.;


> This committee suggests that the Antu cluster should be relocated to the summit, even if it means reducing the cluster size in Chile and increasing the capability at the USDF.